goal_src/jak2/engine/draw/drawable-group-h.gc

```diff
@@ -5,30 +5,16 @@
 ;; name in dgo: drawable-group-h
 ;; dgos: ENGINE, GAME
 
+;; DECOMP BEGINS
+
 (deftype drawable-group (drawable)
   ((length int16             :offset          6)
    (data   drawable :dynamic :offset-assert  32)
    )
   :method-count-assert 17
   :size-assert         #x20
   :flag-assert         #x1100000020
-  )
-
-(defmethod inspect drawable-group ((obj drawable-group))
```

Comment:
this inspect method is actually kinda useful because a bunch of `drawable` types are children of this, and it lets you see what's in them.